### PR TITLE
Fix typo null as the bottom of proto chain

### PR DESCRIPTION
**Pull request by MirzayevOrzu**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by teoli2003** [comment]
I think the original text is correct: the parent of `Object` is `null`. Though the use of _top_ for a prototype chain is a bit unfortunate.

**Comment by MirzayevOrzu** [comment]
Maybe phrase "on top of" can also be removed as it would become "which sits just above null on the prototype chain."

File: files/en-us/web/javascript/inheritance_and_the_prototype_chain/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ JavaScript is a bit confusing for developers experienced in class-based language
 
 When it comes to inheritance, JavaScript only has one construct: objects. Each object has a private property which holds a link to another object called its **prototype**. That prototype object has a prototype of its own, and so on until an object is reached with `null` as its prototype. By definition, `null` has no prototype, and acts as the final link in this **prototype chain**.
 
-Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which sits just below `null` on the top of a prototype chain.
+Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which sits just above `null` on the top of a prototype chain.
```
Comment (teoli2003) [inline files/en-us/web/javascript/inheritance_and_the_prototype_chain/index.md:17]: I agree that the notion of _top_ is not good, but then we can't speak of below and above.

What about:
```suggestion
Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which has `null` as its prototype.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/inheritance_and_the_prototype_chain/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ JavaScript is a bit confusing for developers experienced in class-based language
 
 When it comes to inheritance, JavaScript only has one construct: objects. Each object has a private property which holds a link to another object called its **prototype**. That prototype object has a prototype of its own, and so on until an object is reached with `null` as its prototype. By definition, `null` has no prototype, and acts as the final link in this **prototype chain**.
 
-Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which sits just below `null` on the top of a prototype chain.
+Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which has `null` as its prototype.
```
Comment (wbamberg) [inline files/en-us/web/javascript/inheritance_and_the_prototype_chain/index.md:17]:
```suggestion
Nearly all objects in JavaScript are instances of {{jsxref("Object")}}, which has `null` as its prototype.
```

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**